5.0 Draft/IDD/IDDs Authorization System/eu.arrowhead.authorization-http-json.yml

```diff
@@ -0,0 +1,366 @@
+  x-compression: to-add
+  x-semantics: to-add
+  x-Ontology: to-add
+
+paths:
+
+  /echo:
+    get:  
+      summary: Endpoint to check the core system availability. 
+      description: Returns an OK to indicate that the core system is available. 
+      responses:
+        '200':
+          description: OK
+
+  /authorizations:
+  post:
+    summary: Bulk Authorize access rights for Resources
+    description: |
+      Allows for bulk authorization requests to be sent to the authorization system in a single API request
+
+      Queries the authorization system for authorization decision for each specified access right over the 
+      specified consumer and resource pair. 
+
+      As a reponse, for each request in the request body, an authorization decision will be returned.
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              type: object
+              properties:
+                consumer:
+                  type: string
+                  description: The identifier of the entity making the request. The format of the string is 
+                    dependant on the Authentication mechanism used. (e.g., a certificate string, token, id, etc)
+                access-right:
+                  type: string
+                  description: String identifier for the access right needed to be authorized. These access rights
+                   must be pre-defined within the policy engine used. ("consume", "read", "write", "delete", "update", etc)
+                resource:
+                  type: string
+                  description: The identifier of the resource being accessed, e.g., service instance identifier. 
+              required:
+                - consumer
+                - access-right
+                - resource
+            examples:
+              example1:
+                summary: Three authorization requests from different users to different resources.
+                value:
+                  - consumer: "user123"
+                    access-right: "write"
+                    resource: "eu.arrowhead.tempreadings-http-json@MWtzOXIpaUEsOyEhc1NsTwo"
+                  - consumer: "MWtzOXIpaUEsOyEhc1NsTwo="
+                    access-right: "consume"
+                    resource: "/OutsideTemp"
+                  - consumer: "user789"
+                    access-right: "read"
+                    resource: "91"
+              example2:
+                summary: a single authorization request.
+                value:
+                    consumer: "user234"
+                    access-right: "read"
+                    resource: "document4"                  
+      responses:
+        '200':
+          description: Successful authorization
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string                
```

Comment:
Why not a boolean? I guess a string is a bit more explicit. If using a string, we should add an `enum: ["allowed", "denied"]` to make it clear that no other strings can be included in the response.